packages/gatsby/src/query/queue.js

```diff
@@ -59,6 +59,16 @@ const createDevelopQueue = getRunner => {
   return new Queue(handler, queueOptions)
 }
 
+const createAppropriateQueue = (graphqlRunner, runnerOptions = {}) => {
+  if (process.env.NODE_ENV === `production`) {
```

Comment:
I don't think we should rely on NODE_ENV in this situation, but rather it should be passed from the program.